src/templates/Home/AddColumnModal.test.tsx

```diff
@@ -33,9 +38,9 @@ it("should add a column when pressed", async () => {
 
   expect(onClose).toHaveBeenCalled();
   const columns = useZustand.getState().columns;
-  expect(Object.values(columns)).toHaveLength(1);
-  const columnId = Object.keys(columns)[0];
-  expect(columns[columnId]).toMatchSnapshot(
+  expect(columns).toHaveLength(1);
+  const columnId = columns[0].id;
+  expect(columns.find(column => column.id === columnId)).toMatchSnapshot(
```

Comment:
Isn't this the same as doing `expect(columns[0]).toMatchSnapshot` ? since columnId was set with the id of the first element in columns

Reply:
Fixed